lib/puppet/type/openldap_database.rb

```diff
@@ -167,6 +167,8 @@ def should_to_s(_newvalue)
 
   newproperty(:readonly) do
     desc 'Puts the database into read-only mode.'
+    newvalues(:true, :false)
+    defaultto(:false)
```

Comment:
I guess the new default value is the reason for the CI failue:

> REFERENCE.md is outdated

Please update REFERENCE.md with

```sh-session
$ bundle exec rake strings:generate:reference
```

Add it to the PR and CI should continue.